pkgs/by-name/as/astro-language-server/package.nix

```diff
@@ -0,0 +1,97 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, stdenvNoCC
+, jq
+, moreutils
+, nodePackages
+, cacert
+}:
+
+stdenv.mkDerivation rec {
+  pname = "astro-language-server";
+  version = "2.8.4";
+
+  src = fetchFromGitHub {
+    owner = "withastro";
+    repo = "language-tools";
+    rev = "@astrojs/language-server@${version}";
+    hash = "sha256-pUo8bZPQSYLKE9aZHECXOCxlUjiGRdECkqPClRVVick=";
+  };
+
+  sourceRoot = "${src.name}/packages/language-server";
+
+  pnpm-deps =
+    stdenvNoCC.mkDerivation {
+      pname = "${pname}-pnpm-deps";
+      inherit src version;
+
+      nativeBuildInputs = [
+        jq
+        moreutils
+        nodePackages.pnpm
+        cacert
+      ];
+
+      # https://github.com/NixOS/nixpkgs/blob/de80f1eeac3152c5bbfb1f8891b6414d526bfc54/pkgs/by-name/po/pot/package.nix#L54
+    installPhase = ''
```

Comment:
Nit: indentation is off here and spaces are missing somewhere else, can you format this file with `nixpkgs-fmt`?

Reply:
Formatted